### ISSUE TITLE
Warnings displayed when starting bolt on dev server

### ISSUE THREAD

**Issue by Freffles** [stackblitz-labs/bolt.diy#500]
### Describe the bug

On windows 10. I deleted my bolt folder and started from scratch. Cloned the repo, pnpm install and then pnpm run dev.

The following warning were displayed when I attempted to start bolt via pnpm run dev
 warn  Route discovery/manifest behavior is changing in React Router v7
┃ You can use the `v3_lazyRouteDiscovery` future flag to opt-in early.
┃ -> https://remix.run/docs/en/2.13.1/start/future-flags#v3_lazyRouteDiscovery
┗
 warn  Data fetching is changing to a single fetch in React Router v7
┃ You can use the `v3_singleFetch` future flag to opt-in early.
┃ -> https://remix.run/docs/en/2.13.1/start/future-flags#v3_singleFetch

After the server came up, the following warnings were displayed.
Deprecation Warning: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
1 │ @import './variables.scss';
  │         ^^^^^^^^^^^^^^^^^^
  ╵
    D:\Working\RL_Scripts\bolt\bolt.new-any-llm\app\styles\index.scss 1:9  root stylesheet

Deprecation Warning: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
2 │ @import './z-index.scss';
  │         ^^^^^^^^^^^^^^^^
  ╵
    D:\Working\RL_Scripts\bolt\bolt.new-any-llm\app\styles\index.scss 2:9  root stylesheet

Deprecation Warning: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
3 │ @import './animations.scss';
  │         ^^^^^^^^^^^^^^^^^^^
  ╵
    D:\Working\RL_Scripts\bolt\bolt.new-any-llm\app\styles\index.scss 3:9  root stylesheet

Deprecation Warning: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
4 │ @import './components/terminal.scss';
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
    D:\Working\RL_Scripts\bolt\bolt.new-any-llm\app\styles\index.scss 4:9  root stylesheet

Deprecation Warning: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.

More info and automated migrator: https://sass-lang.com/d/import

  ╷
5 │ @import './components/resize-handle.scss';
  │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  ╵
    D:\Working\RL_Scripts\bolt\bolt.new-any-llm\app\styles\index.scss 5:9  root stylesheet

Warning: 3 repetitive deprecation warnings omitted.
Run in verbose mode to see all warnings.

### Link to the Bolt URL that caused the error

NA

### Steps to reproduce

pnpm run dev

### Expected behavior

Expected no warnings

### Screen Recording / Screenshot

NA

### Platform

Windows 10

### Additional context

NA

**Comment by dustinwloring1988** [comment]
all but one have been fixed if you could take another look for me please and confirm.

**Comment by dark-side1x** [comment]
I am having the same problem

**Comment by Freffles** [comment]
> all but one have been fixed if you could take another look for me please and confirm.

Can confirm, thanks. 
[log.txt](https://github.com/user-attachments/files/17984841/log.txt)
Attached log shows the details. The warning regarding data fetching remains.

Server loaded, when I opened the page there was a flurry of "new dependencies optimized" (bottom of the log) and the server reloaded. This is not new behavior, has been the case for the past week or so, Using Chrome Canary, not sure whether this is a bolt thing or a browser thing. Happy to create a new issue if that is the most appropriate course of action.

**Comment by Freffles** [comment]
I have one back now, so that is 2 total.

> remix vite:dev

 warn  Route discovery/manifest behavior is changing in React Router v7
┃ You can use the `v3_lazyRouteDiscovery` future flag to opt-in early.
┃ -> https://remix.run/docs/en/2.13.1/start/future-flags#v3_lazyRouteDiscovery
┗
 warn  Data fetching is changing to a single fetch in React Router v7
┃ You can use the `v3_singleFetch` future flag to opt-in early.
┃ -> https://remix.run/docs/en/2.13.1/start/future-flags#v3_singleFetch

**Comment by dustinwloring1988** [comment]
v3_singleFetch should be the only one now.

**Comment by xKevIsDev** [comment]
closing this issue as seems to be resolved

**Comment by Freffles** [comment]
Fair enough. A lot has changed, still have 1 warning in the dev startup about singleFetch.
